deployment/node-feature-discovery/values.yaml

```diff
@@ -0,0 +1,207 @@
+image:
+  repository: gcr.io/k8s-staging-nfd/node-feature-discovery
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+  
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+nameOverride: ""
+fullnameOverride: ""
+
+master:
+  replicaCount: 1
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
```

Comment:
Is there any reason we shouldn't enable these, by default, except for the `runAsUser` setting?

Also, `allowPrivilegeEscalation: false`

Reply:
i will try to align the defaults with the templates

Reply:
👍